api/src/main/java/io/opencensus/trace/SpanBuilder.java

```diff
@@ -238,25 +238,17 @@ public final Scope startScopedSpan() {
    *
    * <pre><code>
    * Span span = tracer.spanBuilder("MyRunnableSpan").startSpan();
-   * Runnable newRunnable = tracer.wrap(span, new Runnable() {
-   *    {@literal @}Override
-   *     public void run() {
-   *       try {
-   *         myRunnable.run();
-   *       } finally {
-   *         span.end();
-   *       }
-   *     }
-   * });
+   * Runnable newRunnable = tracer.withSpan(span, myRunnable);
    * newRunnable.run();
+   * span.end();
```

Comment:
Shouldn't span.end() be called in a finally block?

Reply:
done.